### PR TITLE
test(message): add integration test to validate  posted message for EU

### PR DESCRIPTION
**Pull request by brunogenaro**
Fixes https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-160331

Add integration test for `messages` to ensure event data matches an EU user.

To test it directly you can use this command: `npm test -- --packages @webex/plugin-messages`

Screen Shot 2020-08-10 at 10 24 50 AM
 

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js [hunk 1]
```diff
@@ -23,40 +23,71 @@ describe('plugin-messages', function () {
...
-      if (!room) {
-        return Promise.resolve();
-      }
+      webex.rooms.remove(room)
+        .catch((reason) => {
+          console.error('Failed to remove room', reason);
+        });
 
-      return webex.rooms.remove(room)
+      webexEU.rooms.remove(room)
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js:90]:
```suggestion
      webexEU.rooms.remove(roomEU)
```

File: packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js [hunk 2]
```diff
@@ -23,40 +23,71 @@ describe('plugin-messages', function () {
...
+            validateMessage(message, text);
+          });
       }));
 
+    // eslint-disable-next-line consistent-return
     after(() => {
-      if (!room) {
-        return Promise.resolve();
-      }
+      webex.rooms.remove(room)
```
Comment (adamweeks) [inline packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js:85]: Let's keep this async and return a promise all:

```suggestion
      return Promise.all([
        webex.rooms.remove(room),
        webexEU.rooms.remove(roomEU)
      ]);
```

File: packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js (hunk 1 above)
Comment (brunogenaro) [reply packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js:90]: Oops, good catch! Thanks @adamweeks

File: packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js (hunk 2 above)
Comment (brunogenaro) [reply packages/node_modules/@webex/plugin-messages/test/integration/spec/messages.js]: Yeah, that's a good idea! Just updated, thanks @adamweeks

**Pull request merged**